core/domain/story_domain.py

```diff
@@ -962,6 +963,25 @@ def _convert_story_contents_v2_dict_to_v3_dict(cls, story_contents_dict):
             node['description'] = ''
         return story_contents_dict
 
+    @classmethod
+    def _convert_story_contents_v3_dict_to_v4_dict(cls, story_contents_dict):
+        """Converts v3 Story Contents schema to the v4 schema.
+        v4 schema introduces the new schema for Math RTEs.
+
+        Args:
+            story_contents_dict: dict. A dict used to initialize a Story
+                Contents domain object.
+
+        Returns:
+            dict. The converted story_contents_dict.
+        """
+
+        for node in story_contents_dict['nodes']:
+            node['outline'] = (
```

Comment:
@aks681, Why the outine of the story is str. not subtitledHtml object? Note: We won't be able to have a translation for this field in the future.